### ISSUE TITLE
Cannot read property 'options' of undefined

### ISSUE THREAD

**Issue by pgrippi** [gdub22/ember-cli-less#42]
ember-cli-less is used as a dependency in an addon my company uses, which is then used an an addon in an Ember app. With the release of 1.5.0 I am getting this error when running `ember build`:

```
TypeError: Cannot read property 'options' of undefined
  at Class.module.exports.lessOptions (/xxx/node_modules/yyy/node_modules/ember-cli-less/index.js:49:13)
  at LESSPlugin.toTree (/xxx/node_modules/yyy/node_modules/ember-cli-less/index.js:17:32)
  at /xxx/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:186:26
  at Array.forEach (native)
  at processPlugins (/xxx/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:184:11)
  at module.exports.preprocessCss (/xxx/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:154:10)
  at Class.Addon.compileStyles (/xxx/node_modules/ember-cli/lib/models/addon.js:416:12)
  at Class.Addon.treeForAddon (/xxx/node_modules/ember-cli/lib/models/addon.js:397:25)
  at Class._treeFor (/xxx/node_modules/ember-cli/lib/models/addon.js:330:31)
  at Class.treeFor (/xxx/node_modules/ember-cli/lib/models/addon.js:300:19)
```

I don't have any less options specified.

I think this stems from the fix added for https://github.com/gdub22/ember-cli-less/issues/36

**Comment by knownasilya** [comment]
Thanks, will get a fix in asap.

**Comment by knownasilya** [comment]
@pgrippi can you test master? `ember install gdub22/ember-cli-less`

**Comment by pgrippi** [comment]
@knownasilya nope that didn't seem to fix it, it's failing on `app.options` in the if statement

**Comment by pgrippi** [comment]
@knownasilya I put `console.log(app)` above the if statement and it's returning `undefined` in the console

**Comment by knownasilya** [comment]
Ok, added a check for app, give it another try.

**Comment by pgrippi** [comment]
That has fixed it!

**Comment by knownasilya** [comment]
Published as 1.5.1

**Comment by pgrippi** [comment]
@knownasilya thanks so much for the fast turn around :100:
